spec.bs

```diff
@@ -8009,6 +8046,21 @@ A <dfn>direct from seller signals</dfn> is a [=struct=] with the following [=str
     JSON data passed to corresponding buyer's [=script runner=].
 </dl>
 
+<h3 id=additional-bids-structs-section>Additional bids</h3>
+
+A <dfn>signed additional bid with metadata</dfn> is a [=struct=] with the following
+[=struct/items=]:
+
+<dl dfn-for="signed additional bid with metadata">
+  : <dfn>signed additional bid</dfn>
+  :: A [=string=] for an encoded additional bid.
+  : <dfn>seller nonce</dfn>
+  :: Null or a [=string=]. If present, represents the randomly-generated seller nonce returned in
+    the [:Ad-Auction-Additional-Bid:] response header.
+
+    NOTE: This typically will be a [=version 4 UUID=], but the format is not enforced.

```

Comment:
Though we don't enforce that it's a valid UUIDv4, we do enforce that it's 36 characters. Can we use stronger language here, perhaps, "This should be a [=version 4 UUID=}."?

Reply:
That generates a Bikeshed error, as "should" has a specific RFC2119 meaning, and we can't use those in a Note: section, as they're non-normative: 

```
LINE 8693: RFC2119 keyword in non-normative section (use: might, can, has to, or override with <span class=allow-2119>):  This should be a 
<p bs-line-number="8693" class="note"><span class="marker">NOTE:</span> This should be a <a data-link-type="dfn" bs-autolink-syntax="[=version 4 UUID=]" bs-line-number="~8693" data-lt="version 4 UUID" href="https://www.ietf.org/rfc/rfc4122.txt#version-4-uuid" id="ref-for-version-4-uuid⑥" data-silently-dedup="">version 4 UUID</a>, but the format is not enforced.</p>
```

I got rid of the Note: section, and just use must and should in the normative text section, PTAL.